package.json

```diff
@@ -1,5 +1,11 @@
 {
-  "name": "dotcom-scripts",
+  "name": "dotcom",
```

Comment:
am curious if this'll cause issue with `assets/package.json` having the same name. also, are you sure it won't work to define the dependencies in the other package.json file? I'd love to avoid two sets of `node_modules`...

Reply:
It's good to have two separate package declarations. That way, it is clear that the outer one is used for the entire project, and the assets one is used for the assets. I don't think the package's name being the same has any impact. We aren't publishing the package to a registry. The node_modules directories are ignored by docker and git so it doesn't really matter how many there are. Is there any problem with having two other than that there are two?